consensus/processor.go

```diff
@@ -8,12 +8,23 @@ import (
 	"github.com/filecoin-project/go-filecoin/actor"
 	"github.com/filecoin-project/go-filecoin/actor/builtin/account"
 	"github.com/filecoin-project/go-filecoin/address"
+	"github.com/filecoin-project/go-filecoin/metrics"
 	"github.com/filecoin-project/go-filecoin/state"
 	"github.com/filecoin-project/go-filecoin/types"
 	"github.com/filecoin-project/go-filecoin/vm"
 	"github.com/filecoin-project/go-filecoin/vm/errors"
 )
 
+var consensusRegistry *metrics.Registry
+var pbTimer *metrics.Float64Timer
+var amTimer *metrics.Float64Timer
+
+func init() {
+	consensusRegistry = metrics.NewRegistry()
+	amTimer = consensusRegistry.NewTimer("consensus/apply_message", "Duration of message application")
+	pbTimer = consensusRegistry.NewTimer("consensus/process_block", "Duration of block processing")
```

Comment:
Nit: how about camelCase rather than snake_case? Less typing, and idiomatic in many languages except Python.

Reply:
Prometheus prefers sneak case: https://prometheus.io/docs/practices/naming/